docs/jdbc-mysql-catalog.md

```diff
@@ -34,9 +34,10 @@ Gravitino saves some system information in schema and table comment, like `(From
 ### Catalog properties
 
 You can pass to a MySQL data source any property that isn't defined by Gravitino by adding `gravitino.bypass` prefix as a catalog property. For example, catalog property `gravitino.bypass.maxWaitMillis` will pass `maxWaitMillis` to the data source property.
-
 Check the relevant data source configuration in [data source properties](https://commons.apache.org/proper/commons-dbcp/configuration.html)
 
+When you use the Gravitino in Trino. You can pass the Trino MySQL connector configuration using prefix `trino.bypass`. For example using `trino.bypass.join-pushdown.strategy` to passing the `join-pushdown.strategy` to the Gravitino MySQL catalog in Trino runtime.
```

Comment:
"When you use the Gravitino in Trino"

What's the `Gravitino`? Is the Gravitino server or the Gravitino connector?

Reply:
Both server and connector.